.circleci/config.yml

```diff
@@ -2,13 +2,13 @@ version: 2
 jobs:
   build:
     docker:
-    - image: golang:1.9
+    - image: golang:1.13
     working_directory: /go/src/github.com/99designs/gqlgen-contrib
     steps:
     - checkout
-    - run: curl -L -o /bin/dep https://github.com/golang/dep/releases/download/v0.5.0/dep-linux-amd64 && chmod +x /bin/dep
-    - run: go get -u github.com/alecthomas/gometalinter github.com/vektah/gorunpkg && gometalinter --install
-    - run: dep ensure -vendor-only
+    - run: go mod download
+    - run: go get -u github.com/vektah/gorunpkg
```

Comment:
gorunpkg shouldn't be needed with go 1.11+ and modules, `go run github.com/99designs/gqlgen` should work